oak/client/application_client.h

```diff
@@ -59,13 +66,29 @@ class ApplicationClient {
   // Returns a grpc Channel connecting to the specified address, initialised with:
   // - Asylo channel credentials, possibly attesting the entire channel to the remote enclave
   // - Oak Policy call credentials, possibly attaching a specific Oak Policy to each call
```

Comment:
The wording of this confused me for a while; IIUC the channel isn't initialized with call credentials, it's initialized with generator objects that will be invoked to generate per-call credentials (and in our case it just so happens that *every* call will get the same per-call credentials).  Is that right?